spec/models/job_spec.rb

```diff
@@ -54,4 +54,15 @@
       expect(job.author).to eq author
     end
   end
+
+  describe ".search" do
+    let(:matching_title_job) { create(:job, :published, title: "Fullstack developer") }
+    let(:matching_description_job) { create(:job, :published, description: "If you're a fullstack developer, this job is for you!") }
+    let(:not_matching_job) { create(:job, :published, title: "RoR developer", description: "Familiarity with agile methodologies, especially Scrum or Kanban") }
```

Comment:
@sophiedeziel That's why it was on two lines: lines were too long.
What's best then: long lines or CI pleasure? :smile: